### PR TITLE
Added duplication of paths

### PR DESCRIPTION
**Pull request by shueja**

### CODE REVIEW

**Comment by Lewis-Seiden** [comment]
image
disambiguating names does not seem to work properly

File: src/document/PathListStore.ts [hunk 1]
```diff
@@ -44,18 +44,22 @@ export const PathListStore = types
   })
   .actions((self) => {
     return {
-      setActivePathUUID(uuid: string) {
-        if (self.pathUUIDs.includes(uuid)) {
-          self.activePathUUID = uuid;
-        }
-      },
-      addPath(name: string, select: boolean = false): string {
+      disambiguateName(name: string) {
```
Comment (Lewis-Seiden) [inline src/document/PathListStore.ts:47]: image
this does not appear to work properly

**Review by Lewis-Seiden — CHANGES_REQUESTED** [review]

File: src/document/PathListStore.ts (hunk 1 above)
Comment (shueja) [reply src/document/PathListStore.ts:47]: Works as intended, but my intent is probably non-ideal. I wanted a difference between copying "NewPath" twice vs copying NewPath and then copying that copy.

Comment (Lewis-Seiden) [reply src/document/PathListStore.ts:47]: i guess that makes sense, but it does feel a little weird and arbitrary to me without being told that thats the logic behind it

Comment (shueja) [reply src/document/PathListStore.ts:47]: I think it's fine as-is, considering that most people won't be spam-clicking it.

**Review by Lewis-Seiden — APPROVED** [review]

**Pull request merged**